internal/config/config_loader_internal_test.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConfigSearchPath(t *testing.T) {
+	assert := assert.New(t)
+	t.Run("with config file path", func(t *testing.T) {
+		configDirs, configName, ext := buildConfigSearchPath("/something/config/importer.yml")
+		exptectedConfigDirs := append([]string{"/something/config"}, configSearchPaths...)
+		assert.Equal(exptectedConfigDirs, configDirs)
+		assert.Equal("importer", configName)
+		assert.Equal("yml", ext)
+	})
+	
+	t.Run("empty input", func(t *testing.T) {
+		configDirs, configName, ext := buildConfigSearchPath("")
+		assert.Equal(configSearchPaths, configDirs)
+		assert.Equal("", configName)
```

Comment:
Can you add a test for JSON or pick your favorite ext that isn't yaml. 

Also add a use case where  you pass it a file without a the full path like  "testing.yml", which is what most integration tests are doing.